### ISSUE TITLE
Batch Ingested Princeton-Only MVW Scanned Resources are syncing to DPUL as extracted individual volumes

### ISSUE THREAD

**Issue by kelea99** [pulibrary/dpul#1420]
### Summary
Reported by Tom V.  Princeton-Only items have been added to a newly created figgy collection (https://dpul.princeton.edu/korean-textbooks). When they synced over to DPUL, the volumes within MVWs are viewing as individual items (examples of DPUL Vol.s list [here](https://dpul.princeton.edu/korean-textbooks/catalog?utf8=%E2%9C%93&exhibit_id=korean-textbooks&search_field=all_fields&q=Vol.+)) .  Tom tried to reindex, but it failed as just over half. What is weird: the failed DPUL reindex says 66 out of 129 items (see screenshot).  There are 75 items documented in figgy for this collection (50 titles). 

**See Also: https://github.com/pulibrary/dpul/issues/1300

- The materials were batch ingested as MVWs
- There are no examples of Princeton-Only MVWs in EAL to compare this to. 
- There is [one example](https://dpul.princeton.edu/eastasian/catalog/gb19fd96w) of Princeton-Only scanned resource in EAL that is viewing 
- We had problems with scanned resource Princeton Only items in LGBTQAI+ as well.  TICKET: https://github.com/pulibrary/dpul/issues/1402

DPUL collection: https://dpul.princeton.edu/korean-textbooks
Figgy: https://figgy.princeton.edu/?f%5Bmember_of_collection_titles_ssim%5D%5B%5D=Korean+textbook+reprints%3A+1400%27s-1900%27s&per_page=100&sort=title_ssort+desc

### Impact 
_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
There is no rush on this, but Tom wont be able to publish this new DPUL site until the issue is resolved.  
### Priority recommendation

- [ ] asap
- [x] within the next 3 weeks / upon return from winter holiday
- [ ] PO will prioritize

## Sudden Priority Justification
_Required if "asap" or "within the next 3 weeks" is checked.  Add "Sudden Priority" and "Maintenance/Research labels_
this can wait until after the winter holiday
### Expected behaviour
When I go to DPUL, I expect Princeton-Only MVWs to view as MVWs in one viewer, not extracted volumes listed individually. I expect a reindex not to fail and the total numbers reindex to match that of figgy.

### Actual behaviour
When I go to DPUL, i see extracted volumes from MVWs, listed by their "Vol. XX" titles, along with those hooked to MMS ids. Reindex failed and numbers do not add up.

### Screenshots
<img width="1512" alt="Screen Shot 2022-12-14 at 8 16 15 AM" src="https://user-images.githubusercontent.com/45043014/207612634-43b2e2c1-3a0f-4c44-a0a0-317bba90dd80.png">

**Comment by tpendragon** [comment]
Yeah, this is the same bug as Lapidus, where the volumes are in collections and shouldn't be.

The code to fix it will be similar to https://github.com/pulibrary/dpul/issues/1433#issuecomment-1439277200, but we should probably generalize it if it's happening again.

**Comment by tpendragon** [comment]
We stopped this from happening in https://github.com/pulibrary/figgy/pull/5695, but this is a cleanup job.

**Comment by tpendragon** [comment]
Try to fix this for everything that's a child of a scanned resource

**Comment by eliotjordan** [comment]
Removed all child scanned resources from collections (~10,500 records). Will reindex affected collection.

**Comment by eliotjordan** [comment]
@kelea99 This should be fixed. If you find that there are still issues, please re-open.

https://dpul.princeton.edu/korean-textbooks/catalog/dc8049gh066

**Comment by tventimi** [comment]
This is happening again.  Compare

https://figgy.princeton.edu/?f%5Bmember_of_collection_titles_ssim%5D%5B%5D=Korean+textbook+reprints%3A+1400%27s-1900%27s&q=

with

https://dpul.princeton.edu/korean-textbooks/browse/all-exhibit-items

Figgy correctly lists 297 items, DPUL has 337.  (This number actually increased from after attempting a reindex).  The discrepancy is because DPUL has records for individual volumes of the MVWs in this collection.  These have simple names like "Vol. 1", "Vol. 2", etc.

**Comment by tpendragon** [comment]
We found this is true, and it's the same problem as before. Example: https://figgy.princeton.edu/catalog/6b4b1f2b-3485-40e8-a956-3050c927b35c. Both of the volumes have a collection and should not. We thought we made this impossible to happen again here: https://github.com/pulibrary/figgy/pull/5695 

We think maybe bulk ingest is adding collections to volumes (bulk ingesting pdfs?)

## Success Criteria

1. The volumes have no collections associated.
2. Bulk ingest does not add collections to volumes.
   * If this isn't the problem, then find out (with @tventimi probably) how these got ingested so we can identify how they got collections attached to them.

**Comment by tventimi** [comment]
Thanks for looking into this.  Yes, in this case, bulk ingest was used.
